From 91bab84e7a3b7de2591c475ba4912b0db8899f05 Mon Sep 17 00:00:00 2001
From: Mark Vitale <mvitale@sinenomine.net>
Date: Tue, 11 Sep 2018 16:29:59 -0400
Subject: butc: repair build error

Commit c43169fd36348783b1a5a55c5bb05317e86eef82 introduced a build error
by invoking TLog with an extraneous set of internal parentheses.

Remove the offending parentheses.

Change-Id: Ibc52501b01ecbe9f86262566446d63e66486272f
Reviewed-on: https://gerrit.openafs.org/13311
Tested-by: BuildBot <buildbot@rampaginggeek.com>
Reviewed-by: Benjamin Kaduk <kaduk@mit.edu>
---
 src/butc/tcmain.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/butc/tcmain.c b/src/butc/tcmain.c
index c5399396ee1..cbd135b09c2 100644
--- a/src/butc/tcmain.c
+++ b/src/butc/tcmain.c
@@ -835,8 +835,8 @@ tc_IsLocalRealmMatch(void *rock, char *name, char *inst, char *cell)
 
     code = afsconf_IsLocalRealmMatch(dir, &islocal, name, inst, cell);
     if (code) {
-	TLog(0, ("Failed local realm check; code=%d, name=%s, inst=%s, cell=%s\n",
-		 code, name, inst, cell));
+	TLog(0, "Failed local realm check; code=%d, name=%s, inst=%s, cell=%s\n",
+		 code, name, inst, cell);
     }
     return islocal;
 }
-- 
2.18.0

